### PR TITLE
More edits for mvp

### LINKED ISSUE #8: Improve clarity of log-linear transition
Right now, the transition happens pretty quickly and may lose users. Consider:

* Adding explanation of log vs linear scales
* Slowing down the transition
* Improving x-axis labels (esp. for log) and adding units
* Labelling the chart w/ numbers to improve interpretation of the log scale

### LINKED ISSUE #9: Include the uncertainty estimates
Modify the chart to include the uncertainty estimates from Abbot et al. for each feature

### LINKED ISSUE #14: Finalize styling of card
Using default values for now - visual hierarchy and arrangement should be adjusted

### LINKED ISSUE #15: Refine mobile view
Currently x-axis is illegible and y-axis labels dominate view:
image

Could consider something like so:
image

Or something else!

### LINKED ISSUE #25: Add definition of pools and fluxes
Either in text, pop-up, or link

### LINKED ISSUE #26: Add key for colors
Either legend or in figure caption?

### LINKED ISSUE #27: Add link to WSS water cycle page
Add in figure caption or subtitle

### PR DESCRIPTION
**Pull request by hcorson-dosch**
This PR:

* Adds a definition for the terms 'pool' and 'flux' - fixes #25 
* Adds a key for the colors in that definition - fixes #26 
* Sets up a dynamic description of the scale - fixes #8 
* Adds an x-axis label
* Adds a link to the WSS page in the figure caption - fixes #27 
* Finishes styling the dialog cards - fixes #14 
* Refines how the estimates and ranges are provided for the card - fixes #9 
* Drops the mouseover interaction on mobile
* Sets up a mobile view - fixes #15 

Mobile view looks like so:

Log:
image

Linear:
image

Still to work on:
* Fix rendering issue on laptop-sized screens #19 , #21 , #11 
* Final definitions (#25 )
* Update a few images (#3)
* Make accessible to screen readers #22

### CODE REVIEW

**Comment by ceenell** [comment]
I think it would be really useful to be able to see the lines from the y-axis to the points in this view so the points don't appear to be floating. It may also look nice to extend the x-axis to the full width, even though we want to keep that space to the right.

**Comment by hcorson-dosch** [comment]
> It may also look nice to extend the x-axis to the full width, even though we want to keep that space to the right.

Thoughts?

image

**Comment by hcorson-dosch** [comment]
Linear gets crushed:

image

**Comment by ceenell** [comment]
Hmm yeah it's definitely better on the log scale but keeping the same limits on linear is not going to work. Is it a bigger issue to have different axis limits or look imbalanced?

**Comment by hcorson-dosch** [comment]
> I think it would be really useful to be able to see the lines from the y-axis to the points in this view so the points don't appear to be floating.

good idea!

image

**Comment by hcorson-dosch** [comment]
> Hmm yeah it's definitely better on the log scale but keeping the same limits on linear is not going to work. Is it a bigger issue to have different axis limits or look imbalanced?

Hmmm the issue w/ different axis limits is that the margin would need to be different, though we could update that, I suppose. It's a little funny b/c the axis labels go flying off if they are different

File: public/data/pools-fluxes-examples-limited.csv [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
+example,Great Salt Lake,Great Salt Lake,18.9,,,great_salt_lake,cubic kilometers,greatSaltLake.png,"Hannah E. McIlwain | Utah Water Science Center, USGS",,https://www.usgs.gov/media/images/great-salt-lake-may-reach-historic-low-soon,this is a test
+example,Dead Sea,Dead Sea,147,,,dead_sea,cubic kilometers,deadSea.png,https://www.usgs.gov/media/images/kafrein-reservoir-captures-runoff-near-dead-sea-jordan,,https://www.usgs.gov/media/images/kafrein-reservoir-captures-runoff-near-dead-sea-jordan,this is a test
+example,Lake Winnepeg,Lake Winnepeg,284,,,lake_winnepeg,cubic kilometers,lakeWinnepeg.png,"Jacques Descloitres, MODIS Rapid Response Team, NASA/GSFC",,https://visibleearth.nasa.gov/images/72159/lake-winnipeg-manitoba-canada,this is a test
+example,Lake Erie,Lake Erie,480,,,lake_erie,cubic kilometers,lakeErie.png,The Great Lakes: Lake Erie | EPA.gov,,https://www.epa.gov/greatlakes/lake-erie,this is a test
```
Comment (ceenell) [inline public/data/pools-fluxes-examples-limited.csv:30]: Noticing we have no text for the examples.  Do we want to give some context here?

File: src/components/pool_flux_chart.vue [hunk 2]
```diff
@@ -32,14 +33,17 @@
           <label><input type="radio" name="x-scale" value="log" checked> log </label>
           <label><input type="radio" name="x-scale" value="linear"> linear </label>          
         </form>
-        <p>You are probably familiar with linear scales. When you move a fixed distance on a linear axis, you add a fixed value to your starting value. When you move a fixed distance on a log axis, you multiply the starting value by a value of 10.</p>
+        <p :text="axisExplanation">{{ axisExplanation }}</p>
+        <p><span class='poolText' >Pools</span> are places where water is stored, like the ocean. <span class='fluxText'>Fluxes</span> are the ways that water moves between pools, such as evaporation, precipitation, discharge, recharge, or human use.</p>
         <p>The data for this chart are adapted from <a href="https://www.nature.com/articles/s41561-019-0374-y" target="_blank">Abbott et al. (2019) Human domination of the global water cycle absent from depictions and perceptions.</a></p>
+        <p>Learn more about the water cycle and see the water cycle diagram on the <a href="https://www.usgs.gov/water-cycle" target="_blank">USGS Water Science School website.</a></p>
```
Comment (ceenell) [inline src/components/pool_flux_chart.vue:39]: This font size feels very small

File: src/components/pool_flux_chart.vue [hunk 3]
```diff
@@ -32,14 +33,17 @@
           <label><input type="radio" name="x-scale" value="log" checked> log </label>
           <label><input type="radio" name="x-scale" value="linear"> linear </label>          
         </form>
-        <p>You are probably familiar with linear scales. When you move a fixed distance on a linear axis, you add a fixed value to your starting value. When you move a fixed distance on a log axis, you multiply the starting value by a value of 10.</p>
+        <p :text="axisExplanation">{{ axisExplanation }}</p>
+        <p><span class='poolText' >Pools</span> are places where water is stored, like the ocean. <span class='fluxText'>Fluxes</span> are the ways that water moves between pools, such as evaporation, precipitation, discharge, recharge, or human use.</p>
```
Comment (ceenell) [inline src/components/pool_flux_chart.vue:37]: What about moving this text about pools and fluxes to the top under the title?

File: src/components/pool_flux_chart.vue (hunk 2 above)
Comment (hcorson-dosch) [reply src/components/pool_flux_chart.vue:39]: Push a commit that makes it bigger

Comment (ceenell) [reply src/components/pool_flux_chart.vue:39]: It feel like this needs more of an explanation for the scales. "Compare the magnitude of major pools and fluxes of water on Earth. Switch between linear and log scaling of the x-axis using the toggle. Using a log scale is useful when values are distributed across of magnitude. When you move a fixed distance...."

File: src/components/pool_flux_chart.vue (hunk 3 above)
Comment (hcorson-dosch) [reply src/components/pool_flux_chart.vue:37]: I had it there, but wasn't sure if it bumped the chart too far down the page. But now I think it looks nice:
image

I'll push those changes

File: src/components/pool_flux_chart.vue (hunk 2 above)
Comment (ceenell) [inline src/components/pool_flux_chart.vue:39]: The link to the WSS could also be at the top and more prominent, leaving just the scale thing below as a fig caption. That way more people will be likely to click through

File: src/components/pool_flux_chart.vue (hunk 3 above)
Comment (ceenell) [reply src/components/pool_flux_chart.vue:37]: Might also be nice to use the text color for the title, and to make that title bold

File: public/data/pools-fluxes-examples-limited.csv (hunk 1 above)
Comment (hcorson-dosch) [reply public/data/pools-fluxes-examples-limited.csv:30]: Hmm we could add a bit about where they are located? I think that could be in a separate PR.

File: src/components/pool_flux_chart.vue (hunk 2 above)
Comment (hcorson-dosch) [reply src/components/pool_flux_chart.vue]: ^ this would be in the figure caption?

File: src/components/pool_flux_chart.vue [hunk 4]
```diff
@@ -113,20 +121,33 @@ export default {
...
           if (this.showUncertainty) {
-            this.uncertaintyPrompt = 'Hide uncertainty'
+            this.uncertaintyPrompt = 'Hide ranges for estimates'
             this.d3.selectAll('.chartBandBkgd')
               .style("opacity", 1)
             this.d3.selectAll('.chartBand')
               .style("opacity", 0.3)
           } else {
-            this.uncertaintyPrompt = 'Show uncertainty'
+            this.uncertaintyPrompt = 'Show ranges for estimates'
```
Comment (ceenell) [inline src/components/pool_flux_chart.vue:137]: Nowhere are we saying what the default estimate is - mean, median?

Comment (hcorson-dosch) [reply src/components/pool_flux_chart.vue:137]: According to Abbott's paper:
> The central point [this is the e represents the most recent or comprehensive individual estimate, and error bars represent the range of reported values and their uncertainties

File: src/components/pool_flux_chart.vue (hunk 3 above)
Comment (hcorson-dosch) [reply src/components/pool_flux_chart.vue]: Hah I had that and reverted it. I'll add it back in.

File: src/components/pool_flux_chart.vue (hunk 2 above)
Comment (ceenell) [reply src/components/pool_flux_chart.vue]: Something along those lines. We can provide the key info up top and use the space below to explain what's happening with the switching scales, and why it's useful to look at it in log scale, how to interpret.

File: src/components/pool_flux_chart.vue (hunk 4 above)
Comment (ceenell) [reply src/components/pool_flux_chart.vue:137]: interesting......

Comment (ceenell) [reply src/components/pool_flux_chart.vue:137]: definitely need to include that information

**Review by ceenell — APPROVED** [review]
Code looks good, was able to build locally. The text for the examples can wait until later, but please move the text around and add to it a little bit as suggested so the values are clear

File: src/components/pool_flux_chart.vue (hunk 4 above)
Comment (hcorson-dosch) [reply src/components/pool_flux_chart.vue:137]: K added.

**Comment by hcorson-dosch** [comment]
Ok - I've made the changes you requested

**Pull request merged**